app/.env.sample

```diff
@@ -50,4 +49,5 @@ REACT_APP_USER_FORM_HIDDEN_FIELDS=""
 REACT_APP_USER_FORM_RENDER_FIELDS=""
 REACT_APP_PAGINATION_SIZE=1000
 REACT_APP_ENABLE_SERVER_SETTINGS=false
-REACT_APP_OPENSRP_WEB_VERSION="v1.0.0"
+REACT_APP_OPENSRP_WEB_VERSION=$npm_package_version
+REACT_APP_SENTRY_CONFIG_JSON={"dsn":"https://public@sentry.example.com/1","release":"1.0.0","environment":"prod","tags":{"release-name":"web","commit":"012345"}}
```

Comment:
@p-netm do you have an example of how this will look in release files?

Reply:
@machariamuguku we can do so in different ways, the first is to just add it as it looks here as a json, similar to how the roles are added, 
the second,is to add them using yaml object notation, and then add a step in the helmchart to convert it to a json string.

Reply:
@p-netm maybe create an issue for that?